src/main/java/Date.java

```diff
@@ -0,0 +1,67 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Date {
+
+    protected LocalDate date = LocalDate.now();
+    protected int time;
+
+    public Date(String str) throws DukeInvalidDateException {
+        String[] splitted = str.split("\\s+");
```

Comment:
Perhaps you could use a plural form to distinguish a multi-valued variable?

Reply:
Right, I'll probably consider renaming it to `splittedWords` instead. Thank you for your comment.